lib/message.js

```diff
@@ -0,0 +1,61 @@
+import Message from "../models/Message.js";
+
+const createMessage = async (userId, question) => {
+    const response = "Intelligent AI response" //TODO - call AI API
```

Comment:
This brings up a good question. How will we know what course the user wants to query for? 🤔

Reply:
More on this, A.I. will need an identifier to know which index to query